src/prommis/uky/tests/test_uky_flowsheet.py

```diff
@@ -118,433 +131,633 @@ def test_build_flowsheet(self, model):
         assert isinstance(model.fs.sep1_solid, Arc)
         assert isinstance(model.fs.sep1_retained_liquid, Arc)
         assert isinstance(model.fs.sep1_liquid, Arc)
-        assert isinstance(model.fs.recycle1, Arc)
-        assert isinstance(model.fs.purge1, Arc)
-        assert isinstance(model.fs.recycle_feed, Arc)
+        assert isinstance(model.fs.mixed_aq_feed, Arc)
+        assert isinstance(model.fs.org_feed, Arc)
+        assert isinstance(model.fs.mixed_org_feed, Arc)
         assert isinstance(model.fs.s03, Arc)
+        assert isinstance(model.fs.load_recycle, Arc)
         assert isinstance(model.fs.s04, Arc)
         assert isinstance(model.fs.s05, Arc)
         assert isinstance(model.fs.s06, Arc)
+        assert isinstance(model.fs.scrub_recycle, Arc)
         assert isinstance(model.fs.s07, Arc)
         assert isinstance(model.fs.s08, Arc)
+        assert isinstance(model.fs.s09, Arc)
+        assert isinstance(model.fs.s10, Arc)
+        assert isinstance(model.fs.s11, Arc)
+        assert isinstance(model.fs.s12, Arc)
+        assert isinstance(model.fs.s13, Arc)
+        assert isinstance(model.fs.org_feed2, Arc)
+        assert isinstance(model.fs.s14, Arc)
+        assert isinstance(model.fs.s15, Arc)
+        assert isinstance(model.fs.s16, Arc)
+        assert isinstance(model.fs.s17, Arc)
+        assert isinstance(model.fs.s18, Arc)
+        assert isinstance(model.fs.s19, Arc)
+        assert isinstance(model.fs.s20, Arc)
+        assert isinstance(model.fs.s21, Arc)
+        assert isinstance(model.fs.s22, Arc)
+        assert isinstance(model.fs.s23, Arc)
         assert isinstance(model.fs.sep2_solid, Arc)
         assert isinstance(model.fs.sep2_recovered_liquid, Arc)
-        assert isinstance(model.fs.purge2, Arc)
-        assert isinstance(model.fs.recycle2, Arc)
+        assert isinstance(model.fs.s24, Arc)
+        assert isinstance(model.fs.s25, Arc)
 
         assert degrees_of_freedom(model) == 0
 
-    @pytest.mark.unit
-    def test_set_dof(self, model):
-        set_scaling(model)
-        set_operating_conditions(model)
+    @pytest.fixture(scope="class")
+    def scaled_model(self, model):
+        scaled_model = set_scaling(model)
 
-        assert degrees_of_freedom(model) == 0
+        return scaled_model
 
     @pytest.mark.unit
-    def test_initialize_flowsheet(self, model):
-        initialize_system(model)
+    def test_initialize_flowsheet(self, scaled_model):
+        initialize_system(scaled_model)
 
-        assert degrees_of_freedom(model) == 0
+        assert degrees_of_freedom(scaled_model) == 0
 
     @pytest.mark.integration
-    def test_unit_consistency(self, model):
-        assert_units_consistent(model)
+    def test_unit_consistency(self, scaled_model):
+        assert_units_consistent(scaled_model)
```

Comment:
I would suggest replacing this with `assert_no_structural_warnings()` from the diagnostics toolbox.

Reply:
I already have assert_no_strucutural_warnings in a previous test, so I suppose I can just delete this unit check since it's redundant.